modules/pam_timestamp/hmac_openssl_wrapper.c

```diff
@@ -81,22 +86,39 @@ get_crypto_algorithm(pam_handle_t *pamh, int debug){
 }
 
 static int
+PAM_NONNULL((1, 2))
```

Comment:
PAM_NONNULL decorations shouldn't be mixed with other changes.

Reply:
Okay, will split into a separate commit.